assets/tests/hooks/useNearestIntersection.test.ts

```diff
@@ -39,4 +42,39 @@ describe("useNearestIntersection", () => {
     rerender()
     expect(mockNearestIntersection).toHaveBeenCalledTimes(1)
   })
+  const renderUseNearestIntersection = (location: GeographicCoordinate) =>
```

Comment:
**nitpick:** this will scan slightly easier with a space separating it from the `test` block above.

Reply:
Never mind, looks like this is changed in a future commit anyway.